lib/inherited_proofing/mhv/configuration.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'common/client/configuration/rest'
+
+module InheritedProofing
+  module MHV
+    class Configuration < Common::Client::Configuration::REST
+      def valid_id_url
+        Settings.mhv.inherited_proofing.valid_id_url
+      end
+
+      def vacct_info_url
+        Settings.mhv.inherited_proofing.vacct_info_url
+      end
+
+      def app_token
+        Settings.mhv.rx.app_token
+      end
+
+      def base_path
+        Settings.mhv.inherited_proofing.base_path
+      end
+
+      def service_name
+        'mhv_inherited_proofing'
+      end
+
+      def connection
+        @connection ||= Faraday.new(
+          base_path,
+          headers: base_request_headers,
+          request: request_options
+        ) do |conn|
+          conn.use :breakers
+          conn.use Faraday::Response::RaiseError
+          conn.response :snakecase
+          conn.response :json, content_type: /\bjson$/
```

Comment:
We need to add a line to mock this: `conn.response :betamocks if Settings.mhv.inherited_proofing.mock`.

We will then need to add an entry to vets-api-mockdata:
https://github.com/department-of-veterans-affairs/vets-api-mockdata
If you look in that, there is an `mhv` folder, we can add the mocked response in there. I'll be out Monday but I can help on Tuesday if necessary